unittests/tools/test_wpscan_parser.py

```diff
@@ -95,8 +101,52 @@ def test_parse_file_with_multiple_vuln_in_version(self):
             for endpoint in finding.unsaved_endpoints:
                 endpoint.clean()
         self.assertEqual(11, len(findings))
-        finding = findings[2]
-        self.assertEqual("d40374cf-ee95-40b7-9dd5-dbb160b877b1", finding.unique_id_from_tool)
-        self.assertNotEqual("Info", finding.severity)  # it is a vulnerability so not 'Info'
-        self.assertEqual("WordPress 2.8.1-4.7.2 - Control Characters in Redirect URL Validation", finding.title)
-        self.assertEqual("fixed in : 4.6.4", finding.mitigation)
+
+        with self.subTest(i=2):
+            finding = findings[2]
+            self.assertEqual("d40374cf-ee95-40b7-9dd5-dbb160b877b1", finding.unique_id_from_tool)
+            self.assertNotEqual("Info", finding.severity)  # it is a vulnerability so not 'Info'
+            self.assertEqual("WordPress 2.8.1-4.7.2 - Control Characters in Redirect URL Validation", finding.title)
+            self.assertEqual("fixed in : 4.6.4", finding.mitigation)
+            self.assertEqual("Certain", finding.get_scanner_confidence_text())  # data are => 100%
```

Comment:
This finding has no confidence, so the scanner confidence text should not be `Certain`

Reply:
fixed it